projects/packages/sync/src/class-defaults.php

```diff
@@ -148,6 +148,7 @@ class Defaults {
 		'stb_enabled',
 		'stc_enabled',
 		'sm_enabled',
+		'jetpack_subscriptions_subscribe_post_end_enabled',
```

Comment:
There's also a filter you can use to add options to this list directly from module code, but no difference to me which way around we go about it. :-)

https://github.com/Automattic/jetpack/blob/c3fd3c3692624b2a41175a6fe3ddab708209b1cf/projects/packages/sync/src/class-defaults.php#L205-L224

cc @Automattic/jetpack-vulcan in case there's preferences between default list vs filter.

Reply:
Using the filter would be much easier since we wouldn't have to bump the sync package version.

Reply:
I'll try that.